shell/browser/native_window_views.h

```diff
@@ -321,6 +321,9 @@ class NativeWindowViews : public NativeWindow,
   // Handles unhandled keyboard messages coming back from the renderer process.
   views::UnhandledKeyboardEventHandler keyboard_event_handler_;
 
+  // Whether the menubar is visible before fullscreen entered
```

Comment:
Let's tweak this slightly to:

`// Whether the menubar is visible before the window enters fullscreen`